### ISSUE TITLE
fast network camera feature request

### ISSUE THREAD

**Issue by BretG137** [ccrisan/motioneye#2059]
I really appreciate all the hard work on this project and it works great for me.  I would just like to request that fast network camera gets added to motioneye similar to motioneyeos.  I have horrible wifi connection issues with motioneyeos on all my pi zero ws.  Running raspberry pi os and motioneye the connection is rock solid.  The only thing I am missing is the fast network camera feature.  Thanks!!

**Comment by MagicPlants** [comment]
I second this, should be pretty easy right? Or is there an existing feature that does this that we're somehow overlooking? Thanks.

**Comment by starbasessd** [comment]
I don't believe it's possible, because it (motioneyeOS only has enough OS stuff installed to get motion and motionEye running), and even then it disables most of motion and motioneye (using StreamEye) and most of the associated functionality which uses CPU and GPU cycles.

**Comment by MagicPlants** [comment]
Not sure you read our question. We're asking if FNC can be added to Motioneye like it's added to MotioneyeOS. "motioneyeOS only has enough OS stuff installed to get motion and motioneye running" we're not talking about MotioneyeOS. I've been reading through the code and I'm thinking perhaps our solution is just to use streameye and forgo Motioneye altogether.

**Comment by starbasessd** [comment]
The point of Fast Network Camera is to dedicate the Pi to streaming a video stream, and nothing else. You could add StreamEye to raspberry pi OS yourself, and use the 'functionality', just like you can use raspivid. It's here:
https://github.com/ccrisan/streameye
but to get the same 'improvement', you'd have to  turn off or disable most of the rest of the OS functionality.
